src/templates/actors/character-sheet.hbs

```diff
@@ -43,31 +43,141 @@
+              <input name="system.legacy" class="text-entry" type="text" value="{{system.legacy}}" placeholder="{{localize 'sta.actor.character.klingon.legacy'}}" />
+          </div>
+        </div>
+        <div class="row title">
+          <div class="column title" style="width:25%">
+            <div> {{localize 'sta.actor.character.klingon.temperament'}} </div>
+          </div>
+          <div class="column title" style="width:45%">
+              <div> {{localize 'sta.actor.character.klingon.househistory'}} </div>
+          </div>
+          <div class="column title" style="width:10%">
+              <div> {{localize 'sta.actor.character.klingon.influence'}} </div>
+          </div>
+          <div class="column title" style="width:10%">
+              <div> {{localize 'sta.actor.character.klingon.might'}} </div>
+          </div>
+          <div class="column title" style="width:10%">
+              <div> {{localize 'sta.actor.character.klingon.wealth'}} </div>
+          </div>
+        </div>
+        <div class="row data">
+          <div class="column" style="width:25%">
+            <input name="system.temperament" class="text-entry" type="text" value="{{system.temperament}}" placeholder="{{localize 'sta.actor.character.klingon.temperament'}}" />
+          </div>
+          <div class="column" style="width:45%">
+            <input name="system.househistory" class="text-entry" type="text" value="{{system.househistory}}" placeholder="{{localize 'sta.actor.character.klingon.househistory'}}" />
+          </div>
           <div class="column" style="width:10%">
+              <input class="text-entry" name="system.influence" type="number" value="{{system.influence}}" placeholder="0" />
+          </div>
+          <div class="column" style="width:10%">
+              <input class="text-entry" name="system.might" type="number" value="{{system.might}}" placeholder="0" />
+          </div>
+          <div class="column" style="width:10%">
+              <input class="text-entry" name="system.wealth" type="number" value="{{system.wealth}}" placeholder="0" />
+          </div>
+        </div>
+        </div>
+        <div class="row acclaim">
+          <div class="column title" style="width:15%">
+              <div> {{localize 'sta.actor.character.acclaim'}} </div>
+          </div>
+          <div class="column" style="width:5%">
+              <input class="text-entry" name="system.acclaim" type="number" value="{{system.acclaim}}" placeholder="0" />
+          </div>
+          <div class="column title" style="width:15%">
+              <div> {{localize 'sta.actor.character.reprimand'}} </div>
+          </div>
+          <div class="column" style="width:5%">
+              <input class="text-entry" name="system.reprimand" id="currentreprimand" type="number" value="{{system.reprimand}}" placeholder="0" />
+          </div>
+		  <div class="check-button btn3 acclaim" style="width:15%">{{localize 'sta.roll.acclaim'}}</div>
```

Comment:
This is an issue that might be out of bounds of this PR, but the character sheets using divs instead of buttons as buttons is an accessibility issue that should be fixed at some point.  All buttons should be `<buttons>` so that keyboard-only users can get to and use them, and screen reader software can identify their purpose correctly for visually impaired users.

Reply:
Good idea, something to tackle when we do the migration to v13 perhaps as there will be a lot of work required there.